wg-notes/annotations-ucr/index.html

```diff
@@ -311,7 +311,7 @@ <h3>3. Annotation Content</h3>
                     <li>Support for textual notes with rich text formatting for (e.g., bold, italic, underline, subtitles).</li>
                     <li>Support for audio notes.</li>
                     <li>Support for image notes.</li>
-                    <li>Support for an annotation color.</li>
+                    <li>Support for an annotation background and foreground color.</li>
```

Comment:
```suggestion
                    <li>Support for an annotation's highlight color.</li>
```

Reply:
Is this what you meant in https://github.com/w3c/epub-specs/pull/2797#discussion_r2368798260, @llemeurfr ?